cluster/local/kind.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+
+set -e
+
+scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# shellcheck disable=SC1090
+projectdir="${scriptdir}/../.."
+
+# get the build environment variables from the special build.vars target in the main makefile
+eval $(make --no-print-directory -C ${scriptdir}/../.. build.vars)
+
+BUILD_IMAGE="${BUILD_REGISTRY}/${PROJECT_NAME}-amd64"
+DEPLOYMENT_IMAGE="${DOCKER_REGISTRY}/${PROJECT_NAME}:master"
+DEFAULT_NAMESPACE="crossplane-system"
+
+function copy_image_to_cluster() {
+    local build_image=$1
+    local final_image=$2
+    docker tag "${build_image}" "${final_image}"
+    kind load docker-image "${final_image}"
+    echo "Tagged image: ${final_image}"
+}
+
+# Deletes pods with application prefix. Namespace is expected as the first argument
+function delete_pods() {
+    for pod in $(kubectl get pods -n "$2" -l "app=$1" --no-headers -o custom-columns=NAME:.metadata.name); do
+        kubectl delete pod "$pod" -n "$2"
+    done
+}
+
+# current kubectl context == kind-kind, returns boolean
+function check_context() {
+    if [ "$(kubectl config view 2>/dev/null | awk '/current-context/ {print $NF}')" = "kind-kind" ]; then
+        return 0
+    fi
+
+    return 1
+}
+
+# configure kind
+KUBE_IMAGE=${KUBE_IMAGE:-"kindest/node:v1.15.11@sha256:6cc31f3533deb138792db2c7d1ffc36f7456a06f1db5556ad3b6927641016f50"}
+
+case "${1:-}" in
+  up)
+    kind create cluster --image "${KUBE_IMAGE}" --wait 5m
+
+    kubectl apply -f ${scriptdir}/helm-rbac.yaml
+    ${HELM} init --service-account tiller
```

Comment:
Note to self: update `build` with helm 3

Reply:
Actually I though about this and then just made the call that as developers it might be a better idea to use helm2 and kubernetes 1.15 since we're still supporting them so that we can see if our changes break that compatibility. Or maybe it was just finding a reason for procrastination haha.